lib/loader.js

```diff
@@ -82,8 +82,13 @@ module.exports = function(content, map) {
 				"[" + JSON.stringify(importItem.export) + "] + \"";
 		}
 
+		var escapeUrlHelper;
 		cssAsString = cssAsString.replace(result.importItemRegExpG, importItemMatcher.bind(this));
-		if(query.url !== false) {
+		if(query.url !== false && result.urlItems.length > 0) {
+			escapeUrlHelper = "var escapeUrl = require(" +
```

Comment:
```js
escape = "require(" + loaderUtils.stringifyRequest(this, require.resolve("./url/escape.js")) + ")\n"
```

Reply:
I'm a little confused here. Surely you mean `escape = "var escape = require(" + ...`, right?

Don't you think it gets a little confusing when you call both the variable in the loader and in the string by the same name?